### PR TITLE
Added Postgres with *sql.DB

### PR DESCRIPTION
**Pull request by daniil388**

### CODE REVIEW

File: dbump_pgx/pgx.go [hunk 1]
```diff
@@ -27,8 +27,8 @@ func NewMigrator(conn *pgx.Conn) *Migrator {
 // Init migrator.
 func (pg *Migrator) Init(ctx context.Context) error {
 	query := `CREATE TABLE IF NOT EXISTS _dbump_schema_version (
-	version    BIGINT INT NOT NULL PRIMARY KEY,
-	created_at TIMESTAMP NOT NULL
+	version    BIGINT                   NOT NULL PRIMARY KEY,
+	created_at TIMESTAMP WITH TIME ZONE NOT NULL
```
Comment (cristaloleg) [inline dbump_pgx/pgx.go:31]: Why so?

File: dbump_pgx/pgx.go [hunk 2]
```diff
@@ -66,6 +66,8 @@ SET created_at = NOW();`
 
 // Exec is a method for Migrator interface.
 func (pg *Migrator) Exec(ctx context.Context, query string, args ...interface{}) error {
-	_, err := pg.conn.Exec(ctx, query)
-	return err
+	return pg.conn.BeginTxFunc(ctx, pgx.TxOptions{}, func(tx pgx.Tx) error {
+		_, err := tx.Exec(ctx, query, args...)
+		return err
+	})
```
Comment (cristaloleg) [inline dbump_pgx/pgx.go:72]: Let's keep this change separately. There must be a new feature to use tx/not for each migration step.

File: dbump_postgres/postgres.go [hunk 3]
```diff
@@ -0,0 +1,87 @@
+package dbump_postgres
```
Comment (cristaloleg) [inline dbump_postgres/postgres.go:1]: WDYT `dbump_pg` ? :)

**Review by cristaloleg — COMMENTED** [review]
I wonder do we need a go.mod for `dbump_pg` ?

File: dbump_postgres/postgres.go (hunk 3 above)
Comment (cristaloleg) [reply dbump_postgres/postgres.go:1]: Nah, let's leave it as it's.

File: dbump_pgx/pgx.go (hunk 1 above)
Comment (daniil388) [reply dbump_pgx/pgx.go:31]: TIMESTAMP without time zone it is a bug in most cases.
Just a habit :)

https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_timestamp_.28without_time_zone.29

File: dbump_pgx/pgx.go (hunk 2 above)
Comment (daniil388) [reply dbump_pgx/pgx.go]: Okay. Rolled back.

File: dbump_postgres/postgres.go (hunk 3 above)
Comment (daniil388) [reply dbump_postgres/postgres.go]: I renamed it, `dbump_pg` looks better :)

**Review by cristaloleg — APPROVED** [review]

**Pull request merged**